File: src/main/java/fr/tikione/c2e/model/web/TocItem.java
package fr.tikione.c2e.model.web;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

/**
 * TOC item.
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class TocItem {
    
    private String title;
    private String url;
    private List<Article> articles;
}

File: src/main/kotlin/fr/tikione/c2e/model/web/TocItem.kt
package fr.tikione.c2e.model.web

/**
 * TOC item.
 */
class TocItem {

    var title: String? = null
    var url: String? = null
    var articles: List<Article>? = null

    constructor() {
    }

    constructor(title: String?, url: String?, articles: List<Article>?) {
        this.title = title
        this.url = url
        this.articles = articles
    }

    override fun toString(): String {
        return "TocItem(title=$title, url=$url, articles=$articles)"
    }
}
